pkg/ddl/ddl_worker.go

```diff
@@ -434,15 +462,62 @@ func (d *ddl) addBatchDDLJobs2Table(tasks []*limitJobTask) error {
 			logutil.BgLogger().Info("pause user DDL by system successful", zap.String("category", "ddl-upgrading"), zap.Stringer("job", job))
 		}
 
+		if job.Version == model.TiDBDDLV2 {
+			toLocalWorker = true
+			if _, err := job.Encode(true); err != nil {
+				return err
+			}
+		}
+
 		jobTasks = append(jobTasks, job)
 		injectModifyJobArgFailPoint(job)
 	}
 
 	se.SetDiskFullOpt(kvrpcpb.DiskFullOpt_AllowedOnAlmostFull)
 
+	if toLocalWorker {
+		for _, task := range tasks {
+			d.localJobCh <- task
+		}
+		return nil
+	}
 	return errors.Trace(insertDDLJobs2Table(sess.NewSession(se), true, jobTasks...))
 }
 
+// combineBatchJobs combine batch jobs to another batch jobs.
+// currently it only support combine CreateTable to CreateTables.
+func (d *ddl) combineBatchJobs(tasks []*limitJobTask) ([]*limitJobTask, error) {
+	if len(tasks) <= 1 {
```

Comment:
Could we add `tasks[0].job.Version != model.TiDBDDLV2` judgement here？Then we reduce the impact on "v1" tasks early.